coordinator/src/node.rs

```diff
@@ -48,34 +45,35 @@ pub struct NodeSettings {
     // At times, we want to disallow opening new positions (e.g. before
     // scheduled upgrade)
     pub allow_opening_positions: bool,
-    pub max_allowed_tx_fee_rate_when_opening_channel: Option<u32>,
-    // Defines if we want to open jit channels
-    pub jit_channels_enabled: bool,
     /// Defines the sats/vbyte to be used for all transactions within the sub-channel
+    ///
+    /// TODO: What is this?
```

Comment:
this used to be a setting that has been used to set the fee on the contract. This has been replaced with a fee estimation from mempool.

We can remove that setting. It never made much sense in the first place 🙈

Reply:
Would love to remove it, thanks!